packages/pluggableWidgets/gallery-native/src/Gallery.tsx

```diff
@@ -37,6 +112,49 @@ export const Gallery = (props: GalleryProps<GalleryStyle>): ReactElement => {
             hasMoreItems={props.datasource.hasMoreItems ?? false}
             itemRenderer={itemRenderer}
             items={props.datasource.items ?? []}
+            filters={useMemo(
+                () =>
+                    isSortableFilterable ? (
+                        <FilterContext.Provider
+                            value={{
+                                filterDispatcher: prev => {
```

Comment:
typing error

Reply:
I don't see any error could you check it again?

Reply:
prev has implicit any type